### PR TITLE
Misc fixes on buy wizard

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: src/js/wizards/Buy/0_Trade/components/OfferTrade.jsx [hunk 1]
```diff
@@ -105,38 +109,56 @@ class OfferTrade extends Component {
...
         </FormGroup>
         { limitless && <p className="mt-3 limits">
-        Limits: {limitDecimals(minToken)} {asset} to <span id="max-token">{limitDecimals(maxToken)} {asset}</span>
+          <Trans 
+            i18nKey="buyer.offerTrade.limits" values={{
+            min: limitDecimals(minToken),
+            max: limitDecimals(maxToken),
+            symbol: asset
+          }}>
+          Limits: 0 Token to <span id="max-token">N Token</span>
```
Comment (jrainville) [inline src/js/wizards/Buy/0_Trade/components/OfferTrade.jsx]: You forgot to add the values 
```suggestion
          Limits: {{min}} {{symbol}} to <span id="max-token">{{max}} {{symbol}}</span>
```

File: src/js/wizards/Buy/0_Trade/components/OfferTrade.jsx [hunk 2]
```diff
@@ -105,38 +109,56 @@ class OfferTrade extends Component {
...
-            <p className="mt-3 limits">Limits: {limitDecimals(minFiat)} {currency.id} to <span id="max-token">{limitDecimals(maxFiat)} {currency.id}</span></p>
+            { amountGreaterThanBalance && <FormFeedback className="d-block">{t('buyer.offerTrade.amountWarning')}</FormFeedback> }
+            <p className="mt-3 limits">
+              <Trans 
+               i18nKey="buyer.offerTrade.limits" values={{
+                min: limitDecimals(minFiat),
+                max: limitDecimals(maxFiat),
+                symbol: currency.id
+              }}>
+              Limits: 0 USD to <span id="max-token">N USD</span>
```
Comment (jrainville) [inline src/js/wizards/Buy/0_Trade/components/OfferTrade.jsx]: Same here

File: src/js/wizards/Buy/0_Trade/components/OfferTrade.jsx [hunk 3]
```diff
@@ -105,38 +109,56 @@ class OfferTrade extends Component {
...
+      <p className="text-muted">You can think of an arbitrator as a neutral <span className="text-black">judge</span> who, when the trade comes to the <span className="clickable text-black" onClick={this.toggleDisputeDialog}>dispute</span>, will review the trade and resolve the dispute.</p>
+    </Trans>
   </ModalDialog>
-  <ModalDialog display={this.state.displayDisputeDialog} onClose={this.toggleDisputeDialog} buttonText="Ok got it">
+  <ModalDialog display={this.state.displayDisputeDialog} onClose={this.toggleDisputeDialog} buttonText={t('buyer.offerTrade.dialogButton')}>
     <RoundedIcon image={disputeImg} className="mb-2" bgColor="blue" />
-    <h2>Dispute</h2>
-    <p className="text-muted">A dispute is a disagreement on a point of law or fact, a conflict of legal views or of interests between two persons.</p>
+    <h2>{t('buyer.offerTrade.dispute')}</h2>
+    <p className="text-muted">{t('buyer.offerTrade.disputeInfo')}</p>
```
Comment (jrainville) [inline src/js/wizards/Buy/0_Trade/components/OfferTrade.jsx]: Did I completely miss this? I guess I just skipped it by accident

Comment (richard-ramos) [reply src/js/wizards/Buy/0_Trade/components/OfferTrade.jsx]: no big deal. We'll capture anything missing when we create a new language file!

**Pull request merged**